README.md

```diff
@@ -1,12 +1,59 @@
+* Fixed app crash when vehicles database was queried but no vehicles were present
 
-#### Libraries
-* Credit to [Paulo Pires](https://about.me/pires) for his [obd-java-api](https://github.com/pires/obd-java-api).
\ No newline at end of file
+### Known bugs and defects
+* Diagnostic code viewing is not implemented
+* Miles per gallon viewing is not implemente
+
+## Install guide
+### Pre-requisites
+Car Care requires a smartphone running Android OS version 4.0 or higher. For communication between the vehicle's OBD and the smartphone, an OBDII capable bluetooth adapter is required. For installation of the Car Care application onto the smartphone a working installation of Android Studio is necessary.
+### Dependent libraries that must be installed
+Car Care uses [Paulo Pires](https://about.me/pires)'s [obd-java-api](https://github.com/pires/obd-java-api) library.
+All dependent libraries required are included in the Github project. 
+### Download instructions
+The project will be downloaded automatically by Android Studio during the initial setup. It may also be downloaded manually from Github by clicking on the *Clone or Download* button on the Car Care Github main page, then clicking *Download Zip*.
+### Build instructions
+#### If the project was downloaded manually as a zip file
+1. Locate and unzip the downloaded file.
+2. Open Android Studio. On the welcome screen, choose *Open an existing Android Studio project* and navigate to the location of the folder containing the project, then click *OK*. The project will then perform an automatic setup.
+#### If the project will be downloaded automatically through Github
+1. Open Android Studio. On the welcome scren, choose *Check out project from Version Control*, and select *Github* from the drop down menu. 
+2. In the Clone Repository dialog enter `https://github.com/LTT-Gatech/CarCareAndroid.git` in the Git Repository URL field, enter the preferred information for the other fields, then click *Clone*.
+3. When prompted to create a Studio project for the sources you have checked out, select *Yes*.
+4. Choose *Import project from external model* and ensure Gradle is selected below this. In the following dialog, select *Use default Gradle wrapper (recommended)*, then select *Finish*. 
+5. When asked which modules/data to include in the project, ensure *CarCareAndroid (root module)* and *:app* are selected, then click *OK*.
+
+#### After the project is added to Android Studio
+Google services must be enabled for the application by generating a google-services.json file. 
+1. Go to the [Google Services website](https://developers.google.com/mobile/add?platform=android&cntapi=signin&cnturl=https:%2F%2Fdevelopers.google.com%2Fidentity%2Fsign-in%2Fandroid%2Fsign-in%3Fconfigured%3Dtrue&cntlbl=Continue%20Adding%20Sign-In). 
+2. Enter *CarCare* in the App name field, and *com.teamltt.carcare* in the Android package name field, then click on *Continue to Choose and configure services*. 
```

Comment:
Change style *CarCare* to `CarCare` and *com.teamltt.carcare* to `com.teamltt.carcare`. This maintains consistency with step 2 of the previous subsection.